jenkins-plugin/src/test/java/org/jenkinsci/plugins/pipeline/maven/NonProductionGradeDatabaseWarningAdministrativeMonitorIntegrationTest.java

```diff
@@ -69,15 +70,10 @@ public void shouldMitigateComputationWithH2Database(JenkinsRule j) throws Except
         assertThat(GlobalPipelineMavenConfig.get().isDaoInitialized()).isFalse();
         assertThat(monitor.isActivated()).isTrue();
 
-        GlobalPipelineMavenConfig.get().setJdbcUrl(null);
-
-        assertThat(GlobalPipelineMavenConfig.get().isDaoInitialized()).isFalse();
-        assertThat(monitor.isActivated()).isFalse();
-
         GlobalPipelineMavenConfig.get().getDao();
 
         assertThat(GlobalPipelineMavenConfig.get().isDaoInitialized()).isTrue();
-        assertThat(monitor.isActivated()).isFalse();
+        assertThat(monitor.isActivated()).isTrue();
```

Comment:
given you have explicitly called `getDao()` isn;t the following code dead now?
(TBH I have no idea why it would have been doing this 101 times.. - just one should have been enough - esp as it seems to not care other than the monitor is activated and the dai is initialized.)

Reply:
no idea neither... those tests were here.